indico/htdocs/js/indico/jquery/ajaxqbubble.js

```diff
@@ -31,7 +31,9 @@
             url: null,
             method: 'GET',
             cache: false,
-            success: null
+            success: null,
+            onClose: null // callback to invoke after closing the qtip by submitting the inner form. the argument
```

Comment:
it would be better to trigger it even if the form wasn't submitted (with a `null` argument) - like in ajaxdialog